tests/test_data/usapears.org/usapears_3.json

```diff
@@ -0,0 +1,32 @@
+{
+  "author": "Kathy Stephenson",
+  "canonical_url": "https://usapears.org/recipe/pear-burrata-salad/",
+  "site_name": "USA Pears",
+  "host": "usapears.org",
+  "language": "en-US",
+  "title": "Pear Burrata Salad",
+  "ingredients": [
+    "4 cups baby arugula",
+    "2 Anjou pears, thinly sliced",
+    "8 oz burrata cheese (4 mini)",
+    "1/3 cup extra-virgin olive oil",
+    "1/4 cup red onion, finely minced",
+    "3 tablespoons white balsamic vinegar",
+    "2 teaspoons stone ground mustard",
+    "kosher salt",
+    "freshly ground black pepper",
+    "1/4 cup crushed candied walnuts"
+  ],
+  "instructions_list": [
+    "Arrange the arugula on a serving platter. Top with the pears and burrata cheese.",
+    "In a small bowl, whisk together the olive oil, onion, vinegar, and mustard. Season to taste with salt and pepper.",
+    "Drizzle some dressing over the salad and garnish with candied walnuts.",
+    "Serve immediately with additional dressing on the side."
+  ],
+  "yields": "0 servings",
```

Comment:
Hrm.  What's going on here I wonder?

Reply:
Appears to be a bug in `get_yields`; when passed an empty string, it returns zero.  It should probably raise a `ValueError` instead (doesn't make sense to attempt to extract yield information from an empty string).